### ISSUE TITLE
AQL not working in 6.1.0

### ISSUE THREAD

**Issue by dogenkigen** [aerospike/aerospike-tools.docker#17]
Seems like AQL stopped working in 6.1.0. To reproduce run:
```
docker run -it aerospike/aerospike-tools:latest aql
aql: error while loading shared libraries: libreadline.so.7: cannot open shared object file: No such file or directory
```
The issue will only happen if the image was not found in local docker repository and we run it "fresh"

**Comment by jtynerbryan** [comment]
My team is also having this issue, happy to help with details if needed. Thanks

**Comment by mcoberly2** [comment]
Fixed the latest and version 6.1.0:
https://hub.docker.com/repository/docker/aerospike/aerospike-tools/tags?page=1&ordering=last_updated

**Comment by dogenkigen** [comment]
@mcoberly2 works for me. Closing the issue
